### DISCUSSION TITLE
MidiBird

### DISCUSSION THREAD

**Discussion by bealbrown** [shanteacontrols/OpenDeck#287] [Show and tell]
![IMG_0340](https://github.com/shanteacontrols/OpenDeck/assets/6363987/d32bf954-e48b-4044-a872-7f354db04f3f)
![IMG_0278](https://github.com/shanteacontrols/OpenDeck/assets/6363987/47b3d18b-f083-41ef-900c-02f3a9639223)

Just wanted a pcb that had potentiometer soldering footprints right on it, to simplify assembly (with the idea that I'd make a lot) only ended up making 5, but I literally edited some config, compiled the firmware, and it worked "out of the box". Really really impressed with this project and the hard work you've put in to achieve this degree of modularity / adaptability.

Thank you!

**Comment by paradajz** [comment]
Looks great. Thanks for sharing!

**Comment by basst22778** [comment]
is there any possibility to get the PCB files for personal use ? 
i would like to build a controller like yours by myself.
